01-Authenticate/README.md

```diff
@@ -1,17 +1,23 @@
-# PHP Authentication for RS256-Signed Tokens
+# PHP API Authentication Demo
 
-This sample demonstrates how to protect endpoints in a PHP API by verifying an incoming JWT signed by Auth0. The token must be signed with the RS256 algorithm and must be verified against your Auth0 JSON Web Key Set.
+This sample demonstrates how to protect endpoints in a PHP API by verifying an incoming JWT signed by Auth0.
+
+The token must be signed with either the RS256 or HS256 algorithms and must be verified against your Auth0 JSON Web Key Set.
```

Comment:
> The token must be signed with either the RS256 or HS256 algorithms and must be verified against your Auth0 JSON Web Key Set.

This change adds in the detail about HS256, whereas previously I can see from the diff that we only mentioned RS256 - was this incorrect and actually this library can be used with both RS256 and HS256 tokens?

If so, I think this needs to be reworded slightly; the way it reads to me currently is that both HS256 and RS256-signed tokens can be verified with the JWKS, whereas actually only RS256 tokens can be verified in such a way (for HS256 you would verify the token using a locally-stored secret key).

Reply:
Good catch, thanks! Not sure where my head was at with that. I've committed a change to the wording 👌